wasm/wasi/wit/deps/hermes-init/api.wit

```diff
@@ -0,0 +1,13 @@
+/// # Interface to Hermes runtime control flow.
+///
+/// ## Permissions
+///
+/// This API is ONLY Available to Integration Tests and is not normally exposed.
+
+interface api {
+    /// Perform Hermes event queue shutdown.
+    done: func(status: result);
```

Comment:
I wasn't suggesting implementing all of the exit() just pointing to the status code needing to be a `u8`.
I think `done: func(status: u8)` is more than enough for our purposes.